Formula/httpd.rb

```diff
@@ -66,7 +249,19 @@ def install
     end
   end
 
+  def caveats; <<-EOS.undent
+    Using ports 80 and 443 require root privileges to start (8080 and 8443 used by default).
+      * Change "Listen 8080" in #{etc}/httpd/httpd.conf
+      * Change "Listen 8443" in #{etc}/httpd/extra/httpd-ssl.conf
+      * Start with `sudo brew services start httpd` or `sudo apachectl start`
+
+    To use suEXEC, you must manually finish the installation with root privileges. See https://httpd.apache.org/docs/current/suexec.html
+      sudo /bin/bash -c "chown root:_www #{opt_bin}/suexec && chmod 4750 #{opt_bin}/suexec"
+    EOS
+  end
```

Comment:
Would rather skip these caveats and stick this as being on non-root-requiring ports.

Reply:
Let's pass on this for now and split root/suexec stuff into another PR so this one can get merged quicker.